src/main/java/com/ait/lienzo/client/widget/LienzoHandlerManager.java

```diff
@@ -140,9 +143,27 @@ public LienzoHandlerManager(final LienzoPanel lienzo)
 
         m_mediators = m_viewport.getMediators();
 
+        handlerRegistrationManager = new HandlerRegistrationManager();
         addHandlers();
     }
 
+    public void destroy()
+    {
+        if (null != m_mediators)
+        {
+            while (null != m_mediators.pop()) {}
+        }
+        if (null != m_touches)
+        {
+            m_touches.clear();
+        }
+        handlerRegistrationManager.removeHandler();
```

Comment:
Nit picking... ```handlerRegistrationManager.destroy()``` would have possibly been clearer/more consistent with this method name (and would have avoided me having to lookup what ```handerRegistrationManager.removeHandler()``` did - which is just to call ```destroy()``` ;-)